generators/add-action/asset-compute/index.js

```diff
@@ -46,8 +49,11 @@ class AssetComputeGenerator extends ActionGenerator {
         }
       }
     })
+
+    const extFolder = path.dirname(this.actionFolder)
```

Comment:
That's not always the case unfortunately, as actions can be added post init with a folder structure that has been decided by the user. I think a better approach would be `path.dirname(this.configPath)` - configPath is the path to the config file that defines the extension (retrievable from super class)

Reply:
(ext config can redefine `actions` config path to point to another action folder)